src/index.js

```diff
@@ -50,13 +52,15 @@ const Solid = {
   AppRegistration: require('./solid/app-registration'),
   appRegistry: require('./app-registry'),
   auth,
+  tls,
   config: require('../config'),
   currentUser: auth.currentUser.bind(auth),
   identity: require('./identity'),
   login: auth.login.bind(auth),
```

Comment:
I didn't see any tests in `solid-auth-oidc` for the login function, so I don't like the idea of swapping out a tested `client.login` for an untested one.